org.gramps_project.Gramps.metainfo.xml

```diff
@@ -0,0 +1,207 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This ideally should be maintained in Gramps Project repository as it can be used by other distributions/stores.

Reply:
I just noticed it already is; feel free to handle it later on.

Reply:
I had to modify the metainfo.xml to go with flathub, and as I understand it needs to be in a specific place for flathub.  So that is why there is a separate metainfo file being moved in the flatpak manifest.  I am confused about what you want me to do.

Reply:
These files are not specific to Flathub by any means, so any changes done there should be applicable to the one shipped within the project.

Reply:
The current version of Gramps is 5.1.3, but 5.2.0 is due out in a couple of months.  At that time the desktop and appdata file shipped with the project might be changed to work with the flatpak, but there are not currently any plans to make changes to the current release since it was released 4 months ago.  

Changes in the desktop file for this flatpak from the desktop file shipped with the release are:
1. different name
2. different icon name
3. modified executable line
4. different location

The appdata/metainfo file changes are:
1. renaming to fit flathub conventions
2. different location
3. id in line 3 is different
4. launchable in line 192 is different